docs/Makefile

```diff
@@ -46,21 +46,21 @@ apidocs:
 
 examples:
 	mkdir -p "${managed_examples}"
-	rsync -a --exclude-from=.examplesignore ../samples/customer-registry-quickstart/ "${managed_examples}/customer-registry-quickstart/"
+	rsync -a --exclude-from=.examplesignore ../samples/java-customer-registry-quickstart/ "${managed_examples}/java-customer-registry-quickstart/"
 	rsync -a --exclude-from=.examplesignore ../samples/java-valueentity-customer-registry/ "${managed_examples}/java-valueentity-customer-registry/"
 	rsync -a --exclude-from=.examplesignore ../samples/java-eventsourced-customer-registry/ "${managed_examples}/java-eventsourced-customer-registry/"
 	rsync -a --exclude-from=.examplesignore ../samples/java-eventing-shopping-cart/ "${managed_examples}/java-eventing-shopping-cart/"
 	rsync -a --exclude-from=.examplesignore ../samples/java-eventsourced-shopping-cart/ "${managed_examples}/java-eventsourced-shopping-cart/"
 	rsync -a --exclude-from=.examplesignore ../samples/java-replicatedentity-shopping-cart/ "${managed_examples}/java-replicatedentity-shopping-cart/"
 	rsync -a --exclude-from=.examplesignore ../samples/java-valueentity-shopping-cart/ "${managed_examples}/java-valueentity-shopping-cart/"
-	rsync -a --exclude-from=.examplesignore ../samples/replicatedentity-examples/ "${managed_examples}/replicatedentity-examples/"
+	rsync -a --exclude-from=.examplesignore ../samples/java-replicatedentity-examples/ "${managed_examples}/java-replicatedentity-examples/"
 	rsync -a --exclude-from=.examplesignore ../samples/valueentity-counter/ "${managed_examples}/valueentity-counter/"
-	rsync -a --exclude-from=.examplesignore ../samples/fibonacci-action/ "${managed_examples}/fibonacci-action/"
-	rsync -a --exclude-from=.examplesignore ../samples/doc-snippets/ "${managed_examples}/doc-snippets/"
-	rsync -a --exclude-from=.examplesignore ../samples/first-service/ "${managed_examples}/first-service/"
+	rsync -a --exclude-from=.examplesignore ../samples/java-fibonacci-action/ "${managed_examples}/java-fibonacci-action/"
+	rsync -a --exclude-from=.examplesignore ../samples/java-doc-snippets/ "${managed_examples}/java-doc-snippets/"
+	rsync -a --exclude-from=.examplesignore ../samples/java-first-service/ "${managed_examples}/java-first-service/"
 
 bundles:
-	bin/bundle.sh --zip "${managed_attachments}/customer-registry-quickstart.zip" ../samples/customer-registry-quickstart
+	bin/bundle.sh --zip "${managed_attachments}/java-customer-registry-quickstart.zip" ../samples/java-customer-registry-quickstart
```

Comment:
this will probably require an update to the download link from the quickstart adoc page

Reply:
Good catch.  
Done in https://github.com/lightbend/akkaserverless-docs/pull/920